.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,26 @@
+## 📑 Proposed changes
+
+Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.
+
+## 🍱 Types of changes
+
+What types of changes does your code introduce to AEP?
+_Put an `x` in the boxes that apply_
+
+- [ ] Enhancement
+- [ ] [New proposal](https://aep.dev/1#workflow)
+
+## 📋 Your checklist for this pull request
+Please review the [AEP Style and Guidance](https://aep.dev/style-guide) for contributing to this repository.
+
+### General
+- [ ] Ensure that basic [Guidance](https://aep.dev/style-guide#guidance) is met.
+- [ ] Ensure to get the [Referencing AEPs](https://aep.dev/style-guide#referencing-aeps) right.
+
+### Additional for new proposal
```

Comment:
```suggestion
### Additional checklist for a new AEP
```

I think we've agreed upon "AEP" being the nouns - although proposal is not incorrect.